modules/images-configuration-blocked-payload.adoc

```diff
@@ -0,0 +1,45 @@
+//Modules included in the following assemblies
+//
+// * openshift_images/image-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="images-configuration-blocked-payload"]
+
+= Allow blocking of a payload registry
+
+When your mirrors are already configured using a ImageContentSourcePolicy (ICSP) object, you can block upstream payload registries in a disconnected environment. 
+
+.Procedure
+
+. Mirror the payload to a registry in your use case using an ImageContentSourcePolicy (ICSP) object. The following example ICSP mirrors the payload `internal-mirror.io/openshift-payload`.
++
+[source, yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: my-icsp
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - internal-mirror.io/openshift-payload
+    source: quay.io/openshift-payload
```

Comment:
It felt repetitive kinda having `the following example ICSP.` then `Example ICSP object`. So in the second sentence of the first step, I ended with a colon instead of a period, and I think it reads a bit better. But I also wanted to know your thoughts?